lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -72,7 +72,8 @@ def string_to_date(value)
   module_function :string_to_date
 
   def iso8601_datetime(year, month, day, hour, minute)
```

Comment:
I wonder if it would be better to pass the timezone with a default in here as the last parameter? I think you can call  a `new_offset` method per https://ruby-doc.org/stdlib-2.3.1/libdoc/date/rdoc/DateTime.html and just automatically use the current timezone info.

I suspect then you don't need to change the tests (untested locally - just something to think about)

Reply:
Will a user ever choose a TZ other than local/utc? I'm trying to think of a situation where I would specify another timezone.

Reply:
I don't think so - but it may be useful when writing tests. It could be inlined into the method as well.